test/elixir_analyzer/test_suite/take_a_number_deluxe_test.exs

```diff
@@ -171,46 +171,203 @@ defmodule ElixirAnalyzer.TestSuite.TakeANumberDeluxeTest do
+          end
 
+          @impl GenServer
+          def handle_cast(:reset_state, state) do
+          end
+
+          @impl GenServer
+          def handle_info(:timeout, state) do
+          end
+
+          @impl GenServer
+          def handle_info(_, state) do
+          end
+        end,
+        defmodule TakeANumberDeluxe do
+          @impl GenServer
           def init(init_arg) do
-            min_number = Keyword.get(init_arg, :min_number)
-            max_number = Keyword.get(init_arg, :max_number)
-            auto_shutdown_timeout = Keyword.get(init_arg, :auto_shutdown_timeout, :infinity)
+          end
+
+          @impl GenServer
+          def handle_call(:report_state, _from, state) do
+          end
+
+          @impl GenServer
+          def handle_call(:queue_new_number, _from, state) do
+          end
+
+          @impl GenServer
+          def handle_call({:serve_next_queued_number, priority_number}, _from, state) do
+          end
+
+          def handle_cast(:reset_state, state) do
+          end
+
+          @impl GenServer
+          def handle_info(:timeout, state) do
+          end
+
+          @impl GenServer
+          def handle_info(_, state) do
+          end
+        end,
+        defmodule TakeANumberDeluxe do
+          @impl GenServer
+          def init(init_arg) do
+          end
 
-            case TakeANumberDeluxe.State.new(min_number, max_number, auto_shutdown_timeout) do
-              {:ok, state} -> {:ok, state, auto_shutdown_timeout}
-              {:error, error} -> {:stop, error}
-            end
+          @impl GenServer
+          def handle_call(:report_state, _from, state) do
+          end
+
+          @impl GenServer
+          def handle_call(:queue_new_number, _from, state) do
+          end
+
+          @impl GenServer
+          def handle_call({:serve_next_queued_number, priority_number}, _from, state) do
+          end
+
+          @impl GenServer
+          def handle_cast(:reset_state, state) do
+          end
+
+          def handle_info(:timeout, state) do
+          end
+
+          @impl GenServer
+          def handle_info(_, state) do
+          end
+        end,
+        defmodule TakeANumberDeluxe do
+          @impl GenServer
+          def init(init_arg) do
           end
 
           @impl GenServer
           def handle_call(:report_state, _from, state) do
-            {:reply, state, state, state.auto_shutdown_timeout}
+          end
+
+          @impl GenServer
+          def handle_call(:queue_new_number, _from, state) do
+          end
+
+          @impl GenServer
+          def handle_call({:serve_next_queued_number, priority_number}, _from, state) do
+          end
+
+          @impl GenServer
+          def handle_cast(:reset_state, state) do
+          end
+
+          @impl GenServer
+          def handle_info(:timeout, state) do
+          end
+
+          def handle_info(_, state) do
           end
```

Comment:
In each example in this list a different callback is missing `@impl GenServer`, in order from the first one to the last one (here)